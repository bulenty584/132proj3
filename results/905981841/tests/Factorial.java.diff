1,2c1
< --- Error (Explicit) ---
< Cannot invoke "value.Value.toString()" because the return value of "sparrow.visitor.Interpreter.access(token.Identifier)" is null
---
> 3628800
