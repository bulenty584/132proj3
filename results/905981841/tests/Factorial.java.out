Exception in thread "main" java.lang.RuntimeException: Error: v13 = [this_Fac + 0]
Wrong argument type in Load
	at sparrow.visitor.Interpreter.visit(Interpreter.java:230)
	at sparrow.Load.accept(Load.java:18)
	at sparrow.visitor.Interpreter.stepUntilReturn(Interpreter.java:34)
	at sparrow.visitor.Interpreter.visit(Interpreter.java:367)
	at sparrow.Call.accept(Call.java:19)
	at sparrow.visitor.Interpreter.stepUntilReturn(Interpreter.java:34)
	at sparrow.visitor.Interpreter.visit(Interpreter.java:97)
	at sparrow.Program.accept(Program.java:15)
	at Main.main(Main.java:35)
